app/main.py

```diff
@@ -1,3 +1,51 @@
+from typing import Any
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int | float) -> object:
```

Comment:
`other` can be `Distance` according to the task. Fix everywhere, please
```suggestion
    def __add__(self, other: int | float | Distance) -> object:
```